gen/libdir/forbidden.conf

```diff
@@ -75,76 +75,119 @@
 # let dob = data
--safe DpDatadW DpDatasW
+-safe DpDatadW
+# `DpDatasW` is stronger than `Pos*W`.
+#-safe DpDatasW
 
 # let addr = [Exp & R]; (basic-dep; [ADDR]; iico_data+; [Exp & M | Imp & Tag & R | Imp & TTD & R | HU | TLBI | DC.CVAU | IC.IVAU]) & ~same-instance
 # let dob = addr
--safe DpAddrdR DpAddrdW DpAddrsR DpAddrsW
+-safe DpAddrdR DpAddrdW DpAddrsR
+# `DpAddrsW` is stronger than `Pos*W`.
+#-safe DpAddrsW
 
 # let ctrl = [Exp & R]; basic-dep; [BCC]; po
 # let dob = ctrl; [Exp & W | HU | TLBI | DC.CVAU | IC]
-# mcat2config7 aarch64.cat -let dob
--safe DpCtrldW DpCtrlsW
+-safe DpCtrldW
+# `DpCtrlsW` is stronger than `Pos*W`.
+#-safe DpCtrlsW
 
 # let dob = addr; [Exp & M]; po; [Exp & W | HU]
--safe [DpAddrdW PodWW] [DpAddrdW PosWW] [DpAddrsW PodWW] [DpAddrsW PosWW] [DpAddrdR PodRW] [DpAddrdR PosRW] [DpAddrsR PodRW] [DpAddrsR PosRW]
+-safe [DpAddrdW PodWW] [DpAddrsW PodWW] [DpAddrdR PodRW] [DpAddrsR PodRW]
+# `[DpAddrsW PosWW] [DpAddrsR PosRW]` are stronger than `Pos*W` as the former are a read followed by a write to the same location.
+# `[DpAddrdW PosWW] [DpAddrdR PosRW]` can be combined by `diy`
+#-safe [DpAddrsW PosWW] [DpAddrsR PosRW]
 
 # let dob = addr; [Exp & M]; lrs; [Exp & R | Imp & Tag & R]
 # let lrs = [W]; ((po & same-loc) & ~(intervening(W,(po & same-loc)))); [R]
--safe [DpAddrdW Rfi] [DpAddrsW Rfi]
+-safe [DpAddrdW PosWR] [DpAddrsW PosWR]
 
 # let dob = data; [Exp & M]; lrs; [Exp & R | Imp & Tag & R]
 # let lrs = [W]; ((po & same-loc) & ~(intervening(W,(po & same-loc)))); [R]
--safe [DpDatadW Rfi] [DpDatasW Rfi]
+-safe [DpDatadW PosWR] [DpDatasW PosWR]
 
 # let ctrl = [Exp & R]; basic-dep; [BCC]; po
 # let IFB-ob = [Exp & R]; ctrl; [IFB]; po
 # let IFB = ISB | EXC-ENTRY-IFB | EXC-RET-IFB
--safe [DpCtrldR ISB] [DpCtrldW ISB] [DpCtrlsR ISB] [DpCtrlsW ISB]
+-safe [DpCtrldR ISB] [DpCtrldW ISB] [DpCtrlsR ISB]
+# [DpCtrlsW ISB] is stronger than `Pos*W`
+#-safe [DpCtrlsW ISB]
 
 # let IFB-ob = [Exp & R]; pick-ctrl-dep; [IFB]; po
 # let IFB = ISB | EXC-ENTRY-IFB | EXC-RET-IFB
--safe [DpCtrlCseldR ISB] [DpCtrlCseldW ISB] [DpCtrlCselsR ISB] [DpCtrlCselsW ISB]
+-safe [DpCtrlCseldR ISB] [DpCtrlCseldW ISB] [DpCtrlCselsR ISB]
+# [DpCtrlCselsW ISB] is stronger than `Pos*W`
+#-safe [DpCtrlCselsW ISB]
 
 # let IFB-ob = [Exp & R]; addr; [Exp & M]; po; [IFB]; po
--safe [DpAddrdR ISBd**] [DpAddrdR ISBs**] [DpAddrdW ISBd**] [DpAddrdW ISBs**] [DpAddrsR ISBd**] [DpAddrsR ISBs**] [DpAddrsW ISBd**] [DpAddrsW ISBs**]
+-safe [DpAddrdR ISBdRR] [DpAddrdR ISBsRR] [DpAddrdW ISBdWR] [DpAddrsR ISBdRR] [DpAddrsR ISBsRR]
+# `[DpAddrsW ISBsWW] [DpAddrsR ISBsRW]` are stronger than `Pos*W` as the former are a read followed by a write to the same location.
+#-safe [DpAddrsW ISBsWW] [DpAddrsR ISBsRW]
+# `[DpAddr** ISB**W]` is stronger than `[DpAddr** Po**W]`; the latter can be combined by `diy`.
+#-safe [DpAddrdR ISBdRW] [DpAddrdR ISBsRW] [DpAddrdW ISBdWW] [DpAddrdW ISBsWW] [DpAddrsR ISBdRW] [DpAddrsW ISBdWW]
```

Comment:
Is this correct? I think we need 
`[DpAddrdR ISBdRW]`
`[DpAddrdW ISBdWW]`
`[DpAddrsR ISBdRW]`
`[DpAddrsW ISBdWW]`
Not sure if I mislead you to remove them, ISBd** is not covered by anything else AFAICT.

When you tested this by adding this as a `-safe` argument to `diy7` and run them through herd7, did they print the Overlapping-lob flag?